tsconfig.json

```diff
@@ -1,4 +1,12 @@
 {
+    "extends": "./tsconfig-base.json",
```

Comment:
should migrate tests inside `core` or `fortigate-autoscale` instead maybe? not sure....

Reply:
can leave them as is atm. New test cases are needed so we can redo the organization when new test cases are added.